src/dso_api/datasets/management/commands/import_schemas.py

```diff
@@ -1,28 +1,45 @@
 from typing import List, Optional
 
+from amsterdam_schema.types import DatasetSchema
+from amsterdam_schema.utils import schema_defs_from_url
 from django.conf import settings
 from django.core.management import BaseCommand
 
 from dso_api.datasets.models import Dataset
-from amsterdam_schema.types import DatasetSchema
-from amsterdam_schema.utils import schema_defs_from_url
-
 from .create_tables import create_tables
 
 
 class Command(BaseCommand):
     help = "Import all known Amsterdam schema files."
     requires_system_checks = False
 
+    def add_arguments(self, parser):
+        parser.add_argument("schema", nargs="*", help="Local schema files to import")
```

Comment:
Misschien het is beter om 2 aparte commandos te makken. 1 voor URLs en andere voor file/directory laden?

Anders gehele commando heeft 2 doelen.

Reply:
Daar heb ik ook over zitten twijfelen. Tegelijk is het functioneel hetzelfde, alleen de plek waar het schema vandaan gehaald wordt is anders.

Wat mij betreft mag het daarom wel 1 commando blijven, maar hoor graag wat de rest hier van vind.